io/transfer/transfer.go

```diff
@@ -91,5 +101,22 @@ type DataEvent struct {
 
 func (DataEvent) ImplementsEvent() {}
 
+// URLEvent is generated when the app is invoked with a URL. This event is sent to
+// all SchemeOps that match the URL scheme.
+//
+// In order to receive URLEvents, it's necessary to use SchemeOp and the app must
+// register a scheme. The scheme can be registered using gogio, with `-schemes <scheme>`
+// flag.
```

Comment:
This comments needs updating. For example

// URLEvent is generated when the program is invoked with a URL. It is matched
// by [URLFilter].
//
// In order to receive the program must register a scheme. The scheme can be registered
// using gogio, with the `-schemes` flag.